### PR TITLE
Media store main

### PR DESCRIPTION
**Pull request by serugeneris**
### Update to the last branch we were working on.

I'm creating this PR to push the original branch we were currently using separately, thus we can continue pushing into the correct repo from now on.

We have a big amount of changes (I think we are including the ones from the previous [PR](https://github.com/edipoarg/codec/pull/2). I'll fill the PR description once we decide if we're splitting it.

Regarding [1c4502a](https://github.com/edipoarg/codec/pull/3/commits/1c4502ad36b9b5f235df3ccf36e067cb06158593), I've granted more space to the photos:

image

We can discuss whether this is what we want.

### CODE REVIEW

File: src/components/modules/Media.svelte [hunk 1]
```diff
@@ -3,7 +3,7 @@
   import Module from "./Module.svelte";
 </script>
 
-<div class="media">
+<div class="media" style="{$ui_store.media_in_view[0] ? 'height: 110%' : ''}">
```
Comment (xaiki) [inline src/components/modules/Media.svelte:6]: needs it's own commit.

File: src/components/modules/MediaForm.svelte [hunk 2]
```diff
@@ -8,10 +8,10 @@
 </script>
 
 <div class="media-form-container" class:expanded>
-    <div class="module_topbar" on:click={() => {expanded = !!!expanded}}>
-      <div class="module_title text_level1"><div class="arrow">«</div>Options</div>
+    <div class="module_topbar">
+      <div class="module_title text_level1" on:click={() => {expanded = !!!expanded}}><div class="arrow">«</div>Options</div>
     </div>
-    <div class="media-form-conteant" class:expanded>
+    <div class="media-form-content" class:expanded>
```
Comment (xaiki) [inline src/components/modules/MediaForm.svelte:14]: typo, needs it's own commit and be applied as a fixup to the former one.

File: src/components/modules/MediaForm.svelte [hunk 3]
```diff
@@ -25,7 +25,7 @@
                 {:else if key[0] !== '_'}
                     <h3 class="label_text" for={key}>{key}</h3>
                     <textarea name={key} rows=3 bind:value={contentAnalysis[key]}
-                          placeholder="describe what you see in the image in a short sentence" />
+                          placeholder="Describe what you see in the image in a short sentence" />
```
Comment (xaiki) [inline src/components/modules/MediaForm.svelte:28]: typo needs to be split into a fixup commit.

File: src/components/modules/MediaForm.svelte [hunk 4]
```diff
@@ -68,8 +70,10 @@
  .module_topbar .module_title {
      display: flex;
      font-size: 16px;
-     transform: translateX(30px) translateY(25px) rotate(90deg);
+     transform: translateX(35px) translateY(30px) rotate(90deg);
```
Comment (xaiki) [inline src/components/modules/MediaForm.svelte:73]: i'm affraid these px translations will break on different screensize/resolutions/dpi

File: src/components/modules/MediumVideo.svelte [hunk 5]
```diff
@@ -31,14 +31,14 @@
...
-  <SvgOverlay classes={classes} selected={selected} annotations={annotations} src={src.replace(/\.[^.]+$/, ".txt")}>
+  <!-- <SvgOverlay classes={classes} selected={selected} annotations={annotations} src={src.replace(/\.[^.]+$/, ".txt")}> -->
   {#if src.includes("mp4") || src.includes("mov")}
       <video controls muted {src} type="video/mp4" style:height="100%"/>
   {:else if src.includes("png") || src.includes("jpeg") || src.includes("jpg")}
       <!-- svelte-ignore a11y-missing-attribute -->
       <img {src} style:height="100%"/>
     {/if}
-  </SvgOverlay>
+  <!-- </SvgOverlay> -->
```
Comment (xaiki) [inline src/components/modules/MediumVideo.svelte:41]: eeeh ? whuchadoing to my svg dude ?

**Review by xaiki — CHANGES_REQUESTED** [review]
some things need to be split out,
SVG needs to work.

File: src/components/modules/Media.svelte (hunk 1 above)
Comment (serugeneris) [reply src/components/modules/Media.svelte:6]: resolved in new PR

File: src/components/modules/MediaForm.svelte (hunk 3 above)
Comment (serugeneris) [reply src/components/modules/MediaForm.svelte:28]: Resolved in new PR

File: src/components/modules/MediaForm.svelte (hunk 2 above)
Comment (serugeneris) [reply src/components/modules/MediaForm.svelte:14]: Resolved in new PR

**Pull request closed without merging**